src/main/webapp/js/community-page-loader.js

```diff
@@ -0,0 +1,33 @@
+
+/** Fetches users and adds them to the page. */
+function fetchUserList(){
+const url = '/user-list';
```

Comment:
Note: styling. (No need to fix now; we'll fix it when we add an autoformatter next week.)